app/views/devise/registrations/new.html.erb

```diff
@@ -0,0 +1,34 @@
+<h2 class="container_form">Sign up</h2>
+
+<div class="container_form">
+<%= form_for(resource, as: resource_name, url: registration_path(resource_name)) do |f| %>
+  <%= render "devise/shared/error_messages", resource: resource %>
+
+  <div class="field form_input ">
+    <%= f.text_field :name, placeholder: "Name", autocomplete: "name", autofocus: true %>
+  </div>
+
+  <div class="field form_input ">
+    <%= f.email_field :email, placeholder: "Email", autofocus: true, autocomplete: "email" %>
+  </div>
+
+  <div class="field form_input ">
+    <% if @minimum_password_length %>
+    <em>(<%= @minimum_password_length %> characters minimum)</em>
+    <% end %><br />
+    <%= f.password_field :password, placeholder: "Password", autocomplete: "new-password" %>
+  </div>
+
+  <div class="field form_input ">
+    <%= f.password_field :password_confirmation, placeholder: "Confirm password", autocomplete: "new-password" %>
+  </div>
+
+  <div class="actions form_input">
+    <%= f.submit "Sign up" , class: "button"%>
+  </div>
```

Comment:
- [ ] Great job on your implementation so far! 👏 However, I'd like to highlight a crucial aspect mentioned in the project requirements. It's important to ensure that the `confirmation` module is implemented in such a way that when a user creates a new account, they are prompted to confirm their email by clicking on a generated link before gaining access to their account. This step is essential for enhancing the security and usability of the application. Please make sure to address this requirement in your implementation. 

Happy coding! 😊🚀🍀